templates/webapi/main/group_builds.html.ep

```diff
@@ -25,11 +25,13 @@
                 % }
             </span>
         </div>
-        <div class="px-2 align-self-end flex-grow-1">
+        <a data-target="#<%= $group_id %>" href="<%= url_for('tests_overview')->query(@tests_overview_url_data, map { (groupid => $_->{id}) } @{$children});%>" class="px-2 align-self-end flex-grow-1" style="text-decoration: none">
```

Comment:
hm, so this is the same URL that was previously already generated. If we keep it like, the same URL for the complete bar as for the build label itself then we should reuse a variable and not call helpers with duplicated code. Either that or someone has an idea how to make the individual sections clickable while still rendering correctly visually.

Reply:
The cheapest way to make individual sections clickable without disturbing the rendering is to simply add an `onclick` attribute to the relevant elements and handle it in JavaScript. Not nice from an accessibility point of view but for an unimportant feature like acceptable. (You could still at least style the elements to show a pointer as cursor on hover.)

However, I believe it should be possible to style the `a` elements so that they don't affect the rendering. Having the `a` element on the outer level seems wrong; I don't think links are supposed to contain block elements. I would have to experiment myself with the elements to figure it out, though.

Reply:
For accessibility when using `onclick` you could actually assign an aria role: https://w3c.github.io/aria/#link

Reply:
that sounds acceptable as well. So two different options to explore if I understand correctly

Reply:
Okay, I wanted to avoid Javascript for accessibility reasons but I saw some ways to make it work with that. Since the individual sections have classes like "progress-bar progress-bar-passed" it should be manageable.
Can you have both at the same time so if you don't have Javascript, it links like it does now but with Javascript it links the individual sections instead? Or is that too complicated/not possible in HTML?

Reply:
I wouldn't implement both ways. If we decide to require JavaScript than that's it. We already use it extensively on the page anyways.

Reply:
The comment I've pushed uses normal links plus some additional CSS so it looks good.